usr/lib/pkcs11/ep11_stdll/ep11tok.conf

```diff
@@ -34,6 +34,11 @@
 #
 #      CPFILTER /etc/opencryptoki/ep11cpfilter.conf
 #
+# To enable optimization of single part Sign/Verify and Encrypt/Decrypt 
+# operations specify the following option: 
+# 
+#      OPTIMIZE_SINGLE_PART_OPERATIONS
+#
```

Comment:
Why is there a config option at all? Shouldnt onehot sign/verify enc/dec be the default i.e. done if its possible?
If it would be the default one could think of a config option to disable it, but then again is there even a reason one would like to do so ?

Reply:
Defering the Init call to a later time may change the behaviour in regards of return codes. I.e. the init call may no longer return an error, but the error will then apear at the next call, when the defered init call is done. Since this may be seen as a behaviour that is not conform with the PKCS#11 standard, Reinhard wanted to enable this explicitely with an option.